boilerplate/openshift/golang-osd-operator/build-opm-catalog.sh

```diff
@@ -319,6 +319,11 @@ function main() {
     local versions
     # shellcheck disable=SC2207
     versions=($(get_prev_operator_version "$bundle_versions_file"))
+    if [ -z ${versions+x} ]
+    then
+        versions[0]=""
+        versions[1]=""
```

Comment:
or
```suggestion
        versions=("" "")
```
but seriously, don't change this :)

Reply:
Ok :)